samples/sensor/sgp40_sht4x/README.rst

```diff
@@ -0,0 +1,66 @@
+.. _sgp40_sht4x:
+
+SGP40 and SHT4X: High accuracy digital I2C humidity sensor and multipixel gas sensor
+####################################################################################
+
+Description
+***********
+
+This sample application periodically measures the ambient temperature, humidity
+and a raw gas sensor value from an SGP40 and SHT4X device.
+The result is written to the console.
+
+You can choose to use the on-chip T/RH compensation of the SGP40
+by feeding the values measured by the SHT4X into it.
+This is enabled in the Application by default, you can turn it off
+by setting :code:`APP_USE_COMPENSATION=n`.
```

Comment:
```
``CONFIG_APP_USE_COMPENSATION=n``
```

Reply:
fixed it. thanks very much for all the reviewing to you!